### DISCUSSION TITLE
Windows 11 updates (KB5029263 and KB5028948) break task view?

### DISCUSSION THREAD

**Discussion by nag11nag** [valinet/ExplorerPatcher#1806] [General]
Microsoft pushed out the above updates yesterday (I'm running version 22H2, on OS build 22621.2134, with Experience Pack 1000.22659.1000.0). I updated EP the same day (to version 22621.1992.56.1). Everything EXCEPT the Task View is working. That's the interface from the below button on the taskbar.

![2023-08-09_09-17-00](https://github.com/valinet/ExplorerPatcher/assets/15315964/87ac1d70-12c8-474b-bffc-d3fe3d524a96)

It's how I manage my 5 virtual desktops. Now I'm unable to move windows between them. Will there be an update to restore the functionality?

**Comment by Leokuma** [comment]
Same here. <kbd>Ctrl</kbd>+<kbd>Win</kbd>+<kbd>Left</kbd>/<kbd>Right</kbd> still works, though. I believe Valinet will look into it. The only way I found to make it work was to switch back to Win11 taskbar, but I still prefer to use the Win10 taskbar even without Task View.

Volume flyout is not being displayed either BTW.

**Comment by NoxelNyx** [comment]
Also having issues with Task View on 22H2 build 22621.2134. Task View button does not work, nor does Win + Tab. In addition, attempting to open Task View with either method will disable Alt + Tab.

**Comment by nulixxx** [comment]
After updating wth KB5029263 and KB5028948 at 2023.08.09 my notebook (Windows 11 Pro, 22H2, Build 22621.2134) refuses to run Task View also. Windows-Tab and Alt-Tab are without effect. My remote computer at Office (Windows 11 Pro, 22H2, Build 22621.1992) has no such problem , though having been updated in the same way.
Running latest Version of Explorer Patcher 22621.1992.56.1

**Comment by ITDev93** [comment]
Explorer Patcher wouldn't even run for me after those two updates...
I had to uninstall, screen kept flickering and settings wouldn't even open, I am assuming those two updates broke Explorer Patcher?

**Reply by ITDev93** [reply]
![image](https://github.com/valinet/ExplorerPatcher/assets/22062468/12dd60c7-0e39-4f00-9ccb-d73b63cc53af)
Windows 11 taskbar is broken, if I right click on it, ExplorerPatcher crashes...

**Reply by ITDev93** [reply]
Explorer Patcher works fine if you don't right-click on the taskbar, so I can live with that till the fix is released...
Take your time and thank you for what you do for the community...

**Comment by pyrates999** [comment]
It is a known bug.  Please wait for it to get fixed.

**Comment by vippium** [comment]
Just wait...I'll make a PR with all known bugs soon. If you found any other bug, comment them in PR

**Comment by Poopooracoocoo** [comment]
Closing as duplicate of https://github.com/valinet/ExplorerPatcher/issues/1687 and https://github.com/valinet/ExplorerPatcher/issues/1802
